ert3/engine/_sample.py

```diff
@@ -22,6 +22,12 @@ def sample(workspace, parameter_group_name, sample_name, ensemble_size):
             dist_config["input"]["std"],
             index=parameter_group["variables"],
         )
+    elif dist_config["type"] == "uniform":
```

Comment:
Would be good to have a test of this code as well. Currently the sampling tests are located in the integration test of the cli. This is not where they should be, we should make `engine` tests instead and write a more light weight test suite for the cli, but that is probably beyond the scope of this PR. So I suggest you put the tests in the cli test for now...

Reply:
Reminded me that I should make an issue on this: https://github.com/equinor/ert/issues/1226

Reply:
Yes, thanks for pointing out and creating the issue! I will have a look 😄